cmd/entrypoint-wrapper/main.go

```diff
@@ -170,6 +191,40 @@ func copyDir(dst, src string) error {
 	return nil
 }
 
+func waitForFile(path string, timeout time.Duration) error {
```

Comment:
Can we just use https://github.com/kubernetes/test-infra/blob/master/prow/config/agent.go#L140 instead and pass the extra logic in the callback fn?

Reply:
No, because it for some unknown reason does not give the event function… the event.  The way the `entrypoint-wrapper` function is set up, there would be no ambiguity since we guarantee the file does not previously exist and only watch a single file, but making those assumptions implicit would be sloppy programming, IMO.

It would also help very little in reducing this already tiny (~30-line) function, and the acrobatics to set both callbacks up only to then cancel everything on the first event would make it less clear in the end.